tests/e2e/data/client/test_data_obs_client.py

```diff
@@ -253,19 +253,18 @@ def test_augment(self):
         meta = do.discovery(self.test_read_table,
                             keywords=('poverty', ),
                             time=('2010 - 2014', ))
-        dataset = do.augment(self.test_data_table, meta)
+        cartodataframe = do.augment(self.test_data_table, meta)
```

Comment:
just a suggestion, in the same way it's used `df` or `gdf` to refer to pandas DataFrame instances and geopandas GeoDataFrame instances, we could standardize the use of `cdf` as a short variable name for our CartoDataFrame.

Reply:
:+1: